### PR TITLE
Added no-continue rule

### PR DESCRIPTION
**Pull request by Cisplatin**
As requested on https://augur.net/bounties/.

Please let me know if there is anything you can recommend changing to this PR.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -16,15 +16,15 @@ describe("Check Plugin's exported object", () => {
...
+		const { meta, rules } = SoliumSecurityPlugin;
 
 		meta.should.type("object");
 		meta.should.have.size(1);
 		meta.should.have.ownProperty("description");
 		meta.description.should.be.type("string");
 
 		rules.should.be.type("object");
-		rules.should.have.size(8);
+		rules.should.have.size(9);
```
Comment (Cisplatin) [inline test/index.js]: Need to be careful here; if another rule is merged before this one, we need to increment this.

Comment (duaraghav8) [reply test/index.js]: Yes I'm aware of this. I'll take care of resolving the conflicts!

File: test/no-continue.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * @author Simon Hajjar <simon.j.hajjar@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require("solium");
+
+var userConfig = {
+        "rules": {
+                "no-continue": "error"
```
Comment (duaraghav8) [inline test/no-continue.js]: you need to change this to `"security/no-continue": "error"`. Simply writing the rule name means Solium only checks the core rules for this rule. `A/B` means `B` rule from `A` plugin.

Comment (Cisplatin) [reply test/no-continue.js]: Fixed that, thanks!

**Comment by Cisplatin** [comment]
@duaraghav8 Fixed the change you asked for - is there anything else I should change?

**Comment by duaraghav8** [comment]
@Cisplatin All good. Will be merging very soon, just need to keep track of few conflicts & doc changes
Thanks for your contribution:)

**Pull request merged**